### PR TITLE
Fix #6: setup repo

### LINKED ISSUE #6: Setup repo
The base project files are needed.

- README
- CONTRUBUTING
- LICENSE
- .gitignore
- etc....

### PR DESCRIPTION
**Pull request by paul-barry-kenzan**
**Associated Issue:**

#6 

**Changes made:**

- Add root-level repo files

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{
```
Comment (thescientist13) [inline package.json]: do we need to set the engine if we are going to use ES6 features?

**Comment by thescientist13** [comment]
ok to test

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
+# Contributing to pipeline-test-node
```
Comment (thescientist13) [inline CONTRIBUTING.md]: this should follow other projects that just link to Keystone's CONTRIBUTING.md
https://github.com/kenzanlabs/pipeline-validate-js/blob/master/CONTRIBUTING.md

Comment (paul-barry-kenzan) [inline CONTRIBUTING.md]: Updated

File: package.json (hunk 1 above)
Comment (paul-barry-kenzan) [inline package.json]: Added

**Pull request merged**